crates/arroyo-connectors/src/kafka/profile.json

```diff
@@ -106,6 +106,16 @@
                     "sensitive": ["apiSecret"]
                 }
             ]
+        },
+        "connectionProperties": {
```

Comment:
This would be a bit better as  as map instead of a vec so that users don't need to know the k/v syntax. This is how we do property overrides on the table:

https://github.com/ArroyoSystems/arroyo/blob/a9ac8f70e6468d9dadf4d7dfd0d7fab01c14f6f6/crates/arroyo-connectors/src/kafka/table.json#L83-L90

Reply:
👍 agreed map is better